lightningd/plugin_hook.h

```diff
@@ -132,8 +132,10 @@ bool plugin_hook_unregister(struct plugin *plugin, const char *method);
 /* Unregister all hooks a plugin has registered for */
 void plugin_hook_unregister_all(struct plugin *plugin);
 
-/* Special sync plugin hook for db: changes[] are SQL statements, with optional
+/* A sync plugin hook for db: changes[] are SQL statements, with optional
  * final command appended. */
-void plugin_hook_db_sync(struct db *db, const char **changes, const char *final);
+UNNEEDED void plugin_hook_call_db_write(struct db *db,
```

Comment:
Is UNNEEDED necessary on this declaration?  Seems odd...

Reply:
Yes I might not need `UNNEEDED`. I dumbly copied the definition's signature from the macro.

Reply:
> `UNNEEDED` seems unneeded

Recursion depth exceeded.